packages/office-ui-fabric-react/src/components/Slider/Slider.types.ts

```diff
@@ -99,17 +103,11 @@ export interface ISliderProps extends React.ClassAttributes<SliderBase> {
   vertical?: boolean;
 
   /**
-   * Optional flag to render the Slider as disabled.
    * @defaultvalue false
+   * Optional flag to render the Slider as disabled.
```

Comment:
Can you please move back the description above the `@defaultvalue false` tag. It causes `api-extractor` to mark it as undocumented. Not sure why it was even moved 🤔

Reply:
done